src/components/common/Footer.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import {
+  FacebookOutlined,
+  GooglePlusOutlined,
+  TwitterOutlined,
+} from '@ant-design/icons';
+import SSLogo from '../../assets/images/SSLogo.png';
+
+function Footer(props) {
+  return (
+    <div className="footerContainer">
+      <div className="one">
```

Comment:
These container divs are required for the CSS Grid functionality

Reply:
Is it advantageous to use CSS grid over Ant Design's Grid system?